README.md

```diff
@@ -129,6 +130,28 @@ These objects contain all the information needed for a client application using
 
 The value of the MediaRegion parameter in the createMeeting() should ideally be set to the one of the media regions which is closest to the user creating a meeting. An implementation can be found under the topic 'Choosing the nearest media Region' in the [Amazon Chime SDK Media Regions documentation](https://docs.aws.amazon.com/chime/latest/dg/chime-sdk-meetings-regions.html).
 
+### Messaging session
+Create a messaging session in your client application to receive messages from Amazon Chime SDK for Messaging.
+```js
+import * as AWS from 'aws-sdk/global';
+import * as Chime from 'aws-sdk/clients/chime';
+
+import {
+  ConsoleLogger,
+  DefaultMessagingSession,
+  LogLevel,
+  MessagingSessionConfiguration,
+} from 'amazon-chime-sdk-js';
+
+this.logger = new ConsoleLogger('SDK', LogLevel.INFO);
```

Comment:
- Recommend having local variables over `this`: `this.logger =` => `const logger =` (as in your previous revision)
- `SDK` => `MyLogger`